app/src/common/shared/com/igalia/wolvic/ui/widgets/prompts/FilePromptWidget.java

```diff
@@ -82,12 +81,14 @@ protected void initialize(Context aContext) {
             }
 
             mPromptDelegate.dismiss();
+            hide(REMOVE_WIDGET);
         });
 
         mUploadButton = findViewById(R.id.positiveButton);
         mUploadButton.setOnClickListener(view -> {
             // TODO upload the file
             mPromptDelegate.dismiss();
+            hide(REMOVE_WIDGET);
```

Comment:
What are those for?

Reply:
`hide(REMOVE_WIDGET)` closes and removes this dialog.